packages/@guidesmiths/cuckoojs-schematics/src/helm/rule.builder.ts

```diff
@@ -0,0 +1,60 @@
+import {
+	apply,
+	MergeStrategy,
+	mergeWith,
+	type Rule,
+	SchematicsException,
+	template,
+	type Tree,
+	url,
+} from '@angular-devkit/schematics';
+import {join} from 'path';
+
+type PackageJson = {
+	scripts: Record<string, string>;
+};
+
+export class RuleBuilder {
+	constructor(
+		private readonly filePath: string,
+		private readonly helmChartName: string) {
+		this.filePath = filePath;
+		this.helmChartName = helmChartName;
+	}
+
+	buildPackageJson = (packageJsonBuffer: Buffer, serviceName: string, tree: Tree) => {
+		const packageJsonString = packageJsonBuffer.toString();
+		const packageJsonObject: PackageJson = JSON.parse(packageJsonString) as PackageJson;
+
+		packageJsonObject.scripts = {
+			...packageJsonObject.scripts,
+			// eslint-disable-next-line @typescript-eslint/naming-convention
+			'helm:upgrade': `helm upgrade -f ./.helm/values.yml ${serviceName} ${this.helmChartName}`,
```

Comment:
We should check if there is already a script with that name to avoid overriding it.